python_tests/pytest_opinfos.py

```diff
@@ -108,6 +108,44 @@
 
 """ End Ternary Operations """
 
+""" Start Dynamic Shape Enabling Operations """
+
+dynamic_shapes_ops = []
+
+# TODO: Add correctness testing as noted below
+vector_at_opinfo = OpInfo(
+    lambda fd: fd.ops.at,
+    "vector_at",
+    # TODO: Check correctness once there are operators that can consume a Vector
+    sample_input_generator=None,
```

Comment:
Suggestion: You can return from an input generator a `vector` or `tensor` input and a `number` or `list` output.